### PR TITLE
Add rclone page

### LINKED ISSUE #173: Create an rclone page
- How to use rclone to sync two dirs on two machines using IP addresses (i.e. using it with a cloud.rc VM)

1. Generate key pair
2. Install rclone
3. `rclone config`
4. Follow prompts, choosing new remote, then ssh/sftp, then enter the following as they come up:
    - host: <ip address of remote>
    - user: <your-blazerid>
    - key_file: <your-private-keyfile>
    - key_file_pass: <your-private-keyfile-password>

- How to use rclone with Cheaha

Same as above, but for host use `cheaha.rc.uab.edu`

- How to use rclone with UAB Box

1. Listing files (lsd)
2. Syncing files (sync)
3. Copying files (copy/copyto)
    - copy retains name
    - copyto allows changing names of single files
4. For everything else follow the docs!

Install on macos: https://rclone.org/install/#macos-installation-with-brew

Install rclone on Windows:

1. Windows Subsystem for Linux (WSL)
    - Install WSL
    - Install linux flavor
    - Install rclone using https://rclone.org/install/#script-installation
6. Windows OS
    - Download from: https://rclone.org/downloads/
    - Place in some memorable location (Perhaps C:/rclone/)
    - Add to PATH (control panel -> system -> Environment Variables... -> User variables -> PATH -> Edit... -> New -> enter the path from second step -> OK)
    - Verify (windows key -> type "cmd" -> press enter -> type "rclone" -> press enter)

### PR DESCRIPTION
**Pull request by wwarriner**
Fixes #173 

- Added an rclone page
- Added a stub for installing software page

### CODE REVIEW

File: docs/data_management/transfer/rclone.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+To copy a single file and change its name, use `rclone copyto <source:><path/oldname> <destination:><path/newname>`.
+
+Example `rclone copyto "C:\users\Name\My Documents\manuscript.docx" box:manuscript\newest.docx`
+
+### Syncing Between Two Devices
+
+To make a destination directory's contents identical to a source directory, use `rclone sync <source:><path> <destination:><path>`
+
+Example: `rclone sync cheaha:"C:\users\Name\My Documents" box:manuscript`.
+
```
Comment (mdefende) [inline docs/data_management/transfer/rclone.md:239]: add warning here that syncing with delete data from the source if it is missing from the destination. original data could be deleted this way

**Comment by mdefende** [comment]
These rclone instructions are more in depth than my rclone usage instructions from pull request #160. However, it is missing the instructions for adding the LTS S3 connection. This seems like a more general page for moving data from Cheaha to various other places. We have plans to add an s3cmd section to the LTS page to compare and contrast with rclone so for now, I think it's better to keep rclone S3 and this rclone separate.

File: docs/data_management/transfer/rclone.md (hunk 1 above)
Comment (wwarriner) [reply docs/data_management/transfer/rclone.md:239]: I had meant to do this but it slipped my mind. Thanks for the reminder!

**Comment by wwarriner** [comment]
That makes sense, having two pages now isn't an issue. I'm aiming for having a general page on various general-use software (like rclone), and then having a specific page or section where needed. We can also cross-link. The specific page can (should?) link to the general page, and the general page can have a stub section referencing the technology that has a section/page.

**Comment by mdefende** [comment]
Yeah, in that case, I think I'll leave the section on LTS on how to set up the connection, but possibly remove the commands section and link to yours? I'm not totally sure yet

**Comment by mdefende** [comment]
I'm going to go ahead and approve this and merge it. Any changes we make should be on my end to make the LTS section fit.

**Review by mdefende — APPROVED** [review]

**Pull request merged**